kornia/color/rgb.py

```diff
@@ -32,7 +76,7 @@ def forward(self, image: torch.Tensor) -> torch.Tensor:  # type: ignore
         return rgb_to_bgr(image)
 
 
-def rgb_to_bgr(image: torch.Tensor) -> torch.Tensor:
+def rgb_to_bgr(image: torch.Tensor) -> torch.Tensor:  # type: ignore
```

Comment:
Same, as far as I know, the # type: ignore is only necessary in the forward call. Avoid it in the functions.